test/cluster/functionality.sh

```diff
@@ -28,6 +28,20 @@
 source "${OS_ROOT}/hack/lib/init.sh"
 os::util::environment::setup_time_vars
 
+# curl_es hits the Elasticsearch API from within a pod.
+function curl_es() {
```

Comment:
This function is a superset of `query_es_from_es` - https://github.com/openshift/origin-aggregated-logging/blob/master/deployer/scripts/util.sh#L406 - were you planning on replacing `query_es_from_es` with this function?